lib/bolt/project.rb

```diff
@@ -159,6 +163,17 @@ def validate
           raise Bolt::ValidationError, "'#{conf}' in bolt-project.yaml must be an array"
         end
       end
+
+      if @data['modules']
+        unless @data['modules'].is_a?(Array)
+          raise Bolt::ValidationError, "modules in bolt-project.yaml must be an array"
```

Comment:
```suggestion
          raise Bolt::ValidationError, "'modules' in bolt-project.yaml must be an array"
```
I'm not sure how to convey the distinction between the modules key and the module declarations themselves.